app/views/content/terms_provider.md

```diff
@@ -1,3 +1,5 @@
+## Terms of use for teacher training providers
```

Comment:
@stevehook Not sure why we’d need to do this? Can we drop commit `b8c809` (‘Split the title’), as this was fine before, and consistent with how we title the other policy content across the service.

Reply:
@paulrobertlloyd sure can drop it, the reason for the change was just me trying to reproduce what was in the google doc:

![image](https://user-images.githubusercontent.com/450843/69719078-a8a43b00-1107-11ea-8fba-ee6b05c3d3bb.png)

Reply:
Dropped that commit, now looks like this...

![image](https://user-images.githubusercontent.com/450843/69719666-e0f84900-1108-11ea-9011-4d752964a9c4.png)

Reply:
Awesome, thanks!